modules/ROOT/pages/policies-policy-types.adoc

```diff
@@ -0,0 +1,49 @@
+= Policy Types
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies are available in different options, such as, Default, Custom, and Automated. Each type of the policy differs by its scope, management, and usability.
+
+== Default Policies
+
+Mulesoft provides several policies that are ready-to-use and cover different areas, such as authentication, security management, threat protection, and tokenization. You can apply any of these policies using the UI to any of your API endpoints. You can also use an API to apply policies.
```

Comment:
More than using the UI, shouldn't it be: Using Anypoint platform? (this should be either UI or API)

Reply:
changed it to "You can apply any of these policies to any of your API endpoints using API Manager in Anypoint Platform." Done! Thanks!